crates/stackable-operator/src/kvp/mod.rs

```diff
@@ -440,28 +268,26 @@ mod test {
             ("stackable.tech/vendor", "Stackable"),
         ]);
 
-        let labels = Labels::try_from(map).unwrap();
+        let labels = Labels::try_from_iter(map).unwrap();
         assert_eq!(labels.len(), 2);
     }
 
     #[test]
-    fn labels_into_map() {
-        let labels = Labels::try_from([
-            ("stackable.tech/managed-by", "stackablectl"),
-            ("stackable.tech/vendor", "Stackable"),
-        ])
-        .unwrap();
+    fn labels_to_unvalidated() {
+        let labels = Labels::from_iter([
+            KeyValuePair::try_from(("stackable.tech/managed-by", "stackablectl")).unwrap(),
+            KeyValuePair::try_from(("stackable.tech/vendor", "Stackable")).unwrap(),
+        ]);
 
-        let map: BTreeMap<String, String> = labels.into();
+        let map: BTreeMap<String, String> = labels.to_unvalidated();
```

Comment:
note: This type annotation is not needed.

Reply:
8f6daa0333b01541fa428d6a98dd49ad3bc85d33